<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-image: url(://Fondo.png);
}

#PuntuacionJ{
color: white;
background-color: #6FCFEB;
qproperty-alignment: AlignCenter;
border: 3px solid #888;
border-radius: 5px;
}

#PuntuacionO{
color: white;
background-color: #6FCFEB;
qproperty-alignment: AlignCenter;
border: 3px solid #888;
border-radius: 5px;
}

#CartaO1{
background-image: url(://trasera.jpg);
}

#CartaO2{
background-image: url(://trasera.jpg);
}

#CartaO3{
background-image: url(://trasera.jpg);
}

#CartaO4{
background-image: url(://trasera.jpg);
}

#CartaO5{
background-image: url(://trasera.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CartaJ1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>530</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaJ2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>530</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaJ3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>530</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaJ4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaJ5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>530</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaO3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaO1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaO4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaO5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaO2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PuntajeJ">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Magneto</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Puntaje</string>
    </property>
   </widget>
   <widget class="QLabel" name="PuntajeO">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Magneto</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Puntaje</string>
    </property>
   </widget>
   <widget class="QLabel" name="PuntuacionO">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Broadway</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QLabel" name="PuntuacionJ">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Broadway</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaCentro9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaCentro6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaCentro4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaCentro1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaCentro3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaCentro2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaCentro5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>260</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaCentro7">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>260</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CartaCentro8">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>260</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
